.env.local

```diff
@@ -0,0 +1,30 @@
+CELERY_TASK_ALWAYS_EAGER=True
+CELERY_TASK_EAGER_PROPAGATES=True
+CELERY_BROKER_URL=memory://
+CELERY_BACKEND=memory
+
+DEBUG=True
+SECRET_KEY=DEV
+
+EMAIL_HOST=LOCALHOST
+EMAIL_HOST_USER=USER
+EMAIL_HOST_PASSWORD=PASSWORD
+EMAIL_PORT=25
+EMAIL_USE_TLS=False
+DEFAULT_FROM_EMAIL=dev@localhost
+SERVER_EMAIL=dev@localhost
+
+SITE_ID=3
```

Comment:
Do you create this elsewhere? Probably remove this as it will break the one created by initcomicdemo